include/core/mir/geometry/dimensions.h

```diff
@@ -29,180 +30,43 @@ namespace mir
 /// and the operations that they support.
 namespace geometry
 {
-
 namespace detail
 {
+// TODO: On next ABI break, switch to just using Wrapper
 template<typename Tag>
-class IntWrapper
+class IntWrapper : public generic::detail::Wrapper<Tag, int>
```

Comment:
Why "public"? Do we really want enable conversions to base?

Reply:
Yes